packages/terracotta-react/src/hooks/README.md

```diff
@@ -0,0 +1,120 @@
+# Main provider hook
+
+The package provides the `useTerracotta` hook that serves the purpose of providing the TerracottaContext values.
+
+## useTerracotta
+
+In order to use the provided hook, make sure that the component that it is being used in is wrapped in the TerracottaContextProvider.
+
+### Usage
+
+```
+import React, { FC } from "react";
+import { useTerracotta } from "@dhi-gras/terracotta-react";
+
+const Component: FC = () => {
+	const { keys, isLoading } = useTerracotta()
+}
+
+```
+
+#### `keys`
+
+An object respective to the response received from the `/keys` endpoint of the providede terracotta host.
```

Comment:
An object respective to the response received from the `/keys` endpoint of the provided terracotta host.